### PR TITLE
Smaller "bandit" docker image

### PR DESCRIPTION
**Pull request by localleon**
This is the solution to #378 . By removing the building dependencies i managed to shrink the image a little bit. The new stats for the image are:

```
REPOSITORY                       TAG                 IMAGE ID            CREATED             SIZE
bandit                           latest              b1cb1623f387        1 second ago       113MB
```

We could combine all the 'RUN' commands into one layer, but I think the lost readability will not be worth the small decrease in size. Unfortunately, the images are not so small due to the runtime dependencies of python

### CODE REVIEW

File: deployments/dockerfiles/bandit/Dockerfile [hunk 1]
```diff
@@ -3,12 +3,11 @@
...
-RUN chmod +x ./jq
-RUN cp jq /usr/bin
+# Installing bandit via pip
+RUN apk add --virtual .build-deps --no-cache git alpine-sdk bash openssh-client \
+    && pip3 install bandit \
+    && apk del .build-deps # remove virtual namespace after building is done 
+
+# Installing https://github.com/stedolan/jq
+RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 \
+    && chmod +x ./jq && cp jq /usr/bin
```
Comment (jairhenrique) [inline deployments/dockerfiles/bandit/Dockerfile]: You can install `jq` with `apk` (https://pkgs.alpinelinux.org/package/edge/main/x86/jq).

Screen Shot 2019-10-02 at 15 00 17

Comment (rafaveira3) [reply deployments/dockerfiles/bandit/Dockerfile]: Awesome, @jairhenrique! Good catch!

Comment (localleon) [reply deployments/dockerfiles/bandit/Dockerfile]: Great, will add that!

**Comment by localleon** [comment]
Since I don't know what's the huskyCI workflow, I've assumed that git was needed building. I will take a look

File: deployments/dockerfiles/bandit/Dockerfile (hunk 1 above)
Comment (localleon) [reply deployments/dockerfiles/bandit/Dockerfile]: After adding jq to apk we are down to 292Mb from the original dockerfile

**Comment by localleon** [comment]
The latest commit should work but it's really near the original size. I'm gonna dig a little bit depper into building images from scratch.

**Comment by localleon** [comment]
Hello @rafaveira3 ,

by using a multi-stage build, i got the image down to **192MB**. I think this is the best we can do on a python image. 

This time i confirmed that the application actually works. I did a test using the following commands
`docker run -it -p 8080:8080 newbandit2 sh`
`git clone --single-branch --branch vulns-Python https://github.com/globocom/huskyCI`
`cd huskyCI && touch results.json` 
`bandit -f json -o results.json main.py 2> /dev/null`
`jq -j -M -c . results.json`

This results in the following json.

**results.json**
```
{ 
   "errors":[ 

   ],
   "generated_at":"2019-10-03T12:23:29Z",
   "metrics":{ 
      "_totals":{ 
         "CONFIDENCE.HIGH":1,
         "CONFIDENCE.LOW":0,
         "CONFIDENCE.MEDIUM":2,
         "CONFIDENCE.UNDEFINED":0,
         "SEVERITY.HIGH":0,
         "SEVERITY.LOW":2,
         "SEVERITY.MEDIUM":1,
         "SEVERITY.UNDEFINED":0,
         "loc":5,
         "nosec":0
      },
      "main.py":{ 
         "CONFIDENCE.HIGH":1,
         "CONFIDENCE.LOW":0,
         "CONFIDENCE.MEDIUM":2,
         "CONFIDENCE.UNDEFINED":0,
         "SEVERITY.HIGH":0,
         "SEVERITY.LOW":2,
         "SEVERITY.MEDIUM":1,
         "SEVERITY.UNDEFINED":0,
         "loc":5,
         "nosec":0
      }
   },
   "results":[ 
      { 
         "code":"1 secret = 'password123!'\n2 \n3 password = 'thisisnotapassword' #nohusky\n4 \n",
         "filename":"main.py",
         "issue_confidence":"MEDIUM",
         "issue_severity":"LOW",
         "issue_text":"Possible hardcoded password: 'password123!'",
         "line_number":1,
         "line_range":[ 
            1,
            2
         ],
         "more_info":"https://bandit.readthedocs.io/en/latest/plugins/b105_hardcoded_password_string.html",
         "test_id":"B105",
         "test_name":"hardcoded_password_string"
      },
      { 
         "code":"2 \n3 password = 'thisisnotapassword' #nohusky\n4 \n5 command = 'print \"this command is not secure!!\"'\n",
         "filename":"main.py",
         "issue_confidence":"MEDIUM",
         "issue_severity":"LOW",
         "issue_text":"Possible hardcoded password: 'thisisnotapassword'",
         "line_number":3,
         "line_range":[ 
            3,
            4
         ],
         "more_info":"https://bandit.readthedocs.io/en/latest/plugins/b105_hardcoded_password_string.html",
         "test_id":"B105",
         "test_name":"hardcoded_password_string"
      },
      { 
         "code":"6 \n7 exec(command)\n8 \n",
         "filename":"main.py",
         "issue_confidence":"HIGH",
         "issue_severity":"MEDIUM",
         "issue_text":"Use of exec detected.",
         "line_number":7,
         "line_range":[ 
            7
         ],
         "more_info":"https://bandit.readthedocs.io/en/latest/plugins/b102_exec_used.html",
         "test_id":"B102",
         "test_name":"exec_used"
      }
   ]
}
```

**Comment by localleon** [comment]
I tried some stuff with git and fucked up really hard. Im so sorry that this happend. I will make a new pull request and move everthing over. Hopefully this is okay

**Pull request closed without merging**